tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -91,12 +91,12 @@ public void RunAllTests()
                 AcquireTokenADFSV4InteractiveFederatedTest,
                 AcquireTokenADFSV4InteractiveNonFederatedTest,
 
-                //B2CFacebookB2CLoginAuthorityAcquireTokenTest,
-                //B2CFacebookMicrosoftAuthorityAcquireTokenTest,
+                B2CFacebookB2CLoginAuthorityAcquireTokenTest,
+                B2CFacebookMicrosoftAuthorityAcquireTokenTest,
                 //B2CGoogleB2CLoginAuthorityAcquireTokenTest,
```

Comment:
yeah, thanks for the link. it's not something I would notice easily in this spot because the lines are commented out thus I can't even hover over these in VS.... and additionally during code reviews this is much less visible, though I could naturally have taken the time to search... minor detail but a text in place would help people as forget some as me (muldvarp with an r)

---
In reply to: [251188931](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/839#discussion_r251188931) [](ancestors = 251188931)

Reply:
sounds good.